frontend/public/components/control-plane-machine-set.tsx

```diff
@@ -0,0 +1,305 @@
+  ...Kebab.getExtensionsActionsForKind(ControlPlaneMachineSetModel),
+  ...Kebab.factory.common,
+];
+const getDesiredReplicas = (resource: ControlPlaneMachineSetKind) => {
+  return resource.spec.replicas;
+};
+const getReadyReplicas = (resource: ControlPlaneMachineSetKind) => {
+  return resource?.status?.readyReplicas || 0;
+};
+
+const ControlPlaneMachineSetCounts: React.FC<ControlPlaneMachineSetCountsProps> = ({
+  resource,
+}) => {
+  const { t } = useTranslation();
+
+  const desiredReplicas = getDesiredReplicas(resource);
+  const replicas = resource?.status?.replicas || 0;
+  const readyReplicas = getReadyReplicas(resource);
+  const updatedReplicas = resource?.status?.updatedReplicas || 0;
+  const unavailableReplicas = resource?.status?.unavailableReplicas || 0;
+
+  return (
+    <PaneBodyGroup>
+      <DescriptionList className="co-detail-table">
+        <Card>
+          <DescriptionListTerm>{t('public~Desired count')}</DescriptionListTerm>
+          <DescriptionListDescription>{`${desiredReplicas}  ${t('public~machines', {
+            count: desiredReplicas,
+          })}`}</DescriptionListDescription>
+        </Card>
+        <Card>
+          <DescriptionListTerm>{t('public~Current count')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <Tooltip content={t('public~The most recently observed number of replicas.')}>
+              <span>{t('public~{{replicas}} machines', { replicas, count: replicas })}</span>
+            </Tooltip>
+          </DescriptionListDescription>
+        </Card>
+        <Card>
+          <DescriptionListTerm>{t('public~Ready count')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <Tooltip
+              content={t('public~The number of ready replicas for this ControlPlaneMachineSet.')}
+            >
+              <span>
+                {t('public~{{readyReplicas}} machines', {
+                  readyReplicas,
+                  count: readyReplicas,
+                })}
+              </span>
+            </Tooltip>
+          </DescriptionListDescription>
+        </Card>
+        <Card>
+          <DescriptionListTerm>{t('public~Updated count')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <Tooltip
+              content={t('public~The number of updated replicas for this ControlPlaneMachineSet.')}
+            >
+              <span>
+                {t('public~{{updatedReplicas}} machines', {
+                  updatedReplicas,
+                  count: updatedReplicas,
+                })}
+              </span>
+            </Tooltip>
+          </DescriptionListDescription>
+        </Card>
+        <Card>
+          <DescriptionListTerm>{t('public~Unavailable count')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <Tooltip
+              content={t(
+                'public~The number of uanavailable replicas for this MachiControlPlaneMachineSetneSet.',
```

Comment:
typo in the tooltip text

Reply:
🤦‍♂️ Good catch, @yapei.  Fixed.

Also noticed `Desired count` did not have a tooltip so I added one.

<img width="389" alt="Screenshot 2025-06-30 at 8 28 42 AM" src="https://github.com/user-attachments/assets/ab54e29b-dad0-4f5a-8dd3-ea71d56e192b" />